tests/model_serving/model_server/authentication/conftest.py

```diff
@@ -26,32 +26,27 @@
 
 @pytest.fixture(scope="session")
 def skip_if_no_authorino_operator(admin_client: DynamicClient):
+    name = "authorino"
```

Comment:
Wouldn't it be better to have this in a constants file? perhaps under something like `class RequiredOperators`

Reply:
at the moment we only check for authorino in this module; it is not mandatory in other use cases so i think it's ok to leave it here for now. wdyt?